extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -20,6 +20,8 @@ import {user} from '../../../src/log';
 import {removeElement} from '../../../src/dom';
 import {timerFor} from '../../../src/timer';
 import {toggle} from '../../../src/style';
+import {waitForRenderStart} from '../../../3p/integration';
```

Comment:
This is a list of ad types that support sending `render-start`.
I add it here because sticky-ad will behave differently. For ad type that supports `render-start` sticky-ad will set background opacity to 1 when ad layoutCallback resolve. For ad type that doesn't support `render-start` sticky-ad will set background opacity to 1 after a fixed amount of time.